src/components/PageBanner/PageBanner.js

```diff
@@ -42,15 +42,15 @@ const PageBanner = props => (
                 Key indicator
               </span>
 
-              <span> {props.indicatorSubCategory}</span>
+              <span> {props.indicatorSubCategory.name}</span>
             </div>
           )}
 
           <IndicatorCard
             size="large"
             className={style.card}
-            color={getColorVariant(props.colorName, '500')}
-            colorName={props.colorName}
+            color={props.indicatorSubCategory && props.indicatorSubCategory.colorName ? getColorVariant(props.indicatorSubCategory.colorName, '500') : getColorVariant(props.colorName, '500')}
```

Comment:
How is this not causing an ESLint max-len error?

This needs to be broken out into a variable for readability (and line length) - so the whole component now needs an explicit return. Also, there's no point in checking for `props.indicatorSubCategory` and `props.indicatorSubCategory.colorName` here since the app would have broken elsewhere (and also for the `colorName` prop below) if there wasn't a `colorName`.

We should expect that if `indicatorSubCategory` is will always have those props (they should both be `.isRequired` in prop types below).

Reply:
"is..."?

This is probably due to context categories having nested categories, but colorName is not present on the subcategory object most of the time (Eg. in "Labour force") so we fall back to the colorName prop. Looks like the indicatorSubCategory check can be removed though 👍